app/src/main/java/com/prangesoftwaresolutions/audioanchor/PlayActivity.java

```diff
@@ -813,6 +799,25 @@ public void onStopTrackingTouch(SeekBar seekBar) {
             }
         });
 
+        final ImageView resetIV = dialogView.findViewById(R.id.reset);
+        resetIV.setOnClickListener(v -> {
+            int speed = Integer.parseInt(getString(R.string.preference_playback_speed_default));
+            float speedFloat = (float) (speed / 10.0);
+            playbackSpeedTV.setText(getResources().getString(R.string.playback_speed_label, speedFloat));
+
+            // Store new playback speed in shared preferences
+            SharedPreferences.Editor editor = mSharedPreferences.edit();
+            editor.putInt(getString(R.string.preference_playback_speed_key), speed);
+            editor.apply();
+
+            // Set position of seeker to default position 1.0
+            playbackSpeedSB.setProgress(5); // default speed is (min+progress)/10 = (5+5)/10 = 1.0
```

Comment:
I think it would be cleaner if you'd use the already existing function `getProgressFromPlaybackSpeed(speed)` here.

Reply:
I was thinking about it. Okey, I'll rewrite these code, using function `getProgressFromPlaybackSpeed(speed)`

Reply:
Exactly! I missed a possibility of use variable `speed`. I have already fixed it.